docker/test/split_build_smoke_test/run.sh

```diff
@@ -4,6 +4,7 @@ set -x
 
 install_and_run_server() {
     mkdir /unpacked
+    cd /unpacked
```

Comment:
Why this change needed?

Reply:
My hunch is that changing the working directory to unpacked my avoid some of the volume mounts off of root directories that could be owned by ubuntu.

Reply:
>Actually, the fix is not good here either

Why not? LGTM.

>I'm not 100% sure what user 1000/ubuntu is owning here.

```sh
$ docker run --rm -it clickhouse/split-build-smoke-test find / -maxdepth 1 \! -user 0
/sys
/proc
```

Obviously.

BTW You may improve logging to show it

This change is required anyway to avoid clickhouse check every directory in the `/`, you can use `clickhouse-server -- --path /unpacked` by the way (plus if you `cd` here, then `tar` does not requires `-C` option)

Reply:
Yes, that's what I suspected but couldn't reproduce it in my default docker setup everything mounts as root. Thank you for the confirmation! I'll definitely try your path suggestion.